### PR TITLE
add new language - simplified Chinese

### PR DESCRIPTION
**Pull request by editor-Ajian**
I am going to contribute translations for bitcointranscript. So I edit config and i8n files in this repo. I can compile successfully in my local machine. Please chack my changes.

### CODE REVIEW

**Review by kouloumos — COMMENTED** [review]
Looks good!🚀 Access to knowledge is often limited in other languages, so transcript translations are appreciated. The only downside is that we can't properly review your translations but hopefully other contributors will eventually find their way to this project and help with reviewing each others work.

Please remove the `.hugo_build.lock` file as is not needed in source control. It should already be in `.gitignore` to avoid this kind of accidental inclusions.

Note: You can easily [preview your transcripts](https://github.com/bitcointranscripts/bitcointranscripts.github.io#preview-your-transcript) (https://github.com/bitcointranscripts/bitcointranscripts/pull/199) on top of this branch with 
```
./preview_branch.sh editor-Ajian master
```
chinese-preview

**Comment by editor-Ajian** [comment]
Got it. What should I do? Open a new PR or just add a commit to this PR (to delete the ` hugo_build.lcok ` )?

By the way, I cannot run the  ` preview_branch.sh ` successfully.

**Comment by kouloumos** [comment]
> Got it. What should I do? Open a new PR or just add a commit to this PR (to delete the `hugo_build.lcok` )?

You can just add a new commit to this PR.

A more advance way would be to [rewrite history](https://git-scm.com/book/en/v2/Git-Tools-Rewriting-History) using `git commit --amend` and then force push (that's the `-f` flag) to your branch (in your case `git push -u origin master -f`). That's the general recommended way when collaborating in repositories in order to keep a clean repository history. That said, GitHub gives us the "Squash and merge" option which results to just one commit regardless of how many commits you will have on your PR.

 merge options 

Also, talking about git, it's a good practice to create a new branch for your changes and not commit directly on the master branch. It's okay for this PR, but on other occasions conflicts might occur. 
To do that; When you have your changes ready, execute `git checkout -b <name-of-your-branch>` (creates a new branch and changes to that one, in general you can change between branches with `git checkout <name-of-branch>`) and then `git push -u origin <name-of-your-branch>`  

> By the way, I cannot run the `preview_branch.sh` successfully.

I don't think that anyone else has ever tested that script so I'm interested to learn more. Can you give more information? What's the error you are getting? What OS are you using?

**Comment by editor-Ajian** [comment]
Hi kouloumos, thanks very much for you advices. They are really useful (as you can see, I am not good at git). 

As the script, I get a message like this:
```
The most similar command is
        remote
Submodule path 'content': checked out 'be8531f274e834c9faa0f8f90f5a4544538f9606'
```
It doesn't return error. It just stays at  ` bitcointranscripts:master ` .

File: i18n/zh.toml [hunk 1]
```diff
@@ -0,0 +1,17 @@
...
+speakers = "演讲者"
+date = "日期"
+
+lead = "有关比特币和闪电网络的演讲记录稿宝库"
+section1_title = "比特币考古学"
+section1_caption= "由 [kanzure](https://github.com/kanzure) 记录的和转移自 [diyhpluswiki](https://github.com/kanzure/diyhpluswiki/tree/master/transcripts) 仓库的重大事件的历史记录。"
+section2_title = "源自社区"
+section2_caption= "请考虑将你转写的文字稿提交到这个 [GitHub 仓库](https://github.com/bitcointranscripts/bitcointranscripts)。"
+section3_title = "按标签浏览"
+section3_caption= "按话题和[标签](/tags)寻找记录稿。"
```
Comment (kouloumos) [inline i18n/zh.toml]:
```suggestion
section3_caption= "按话题和[标签](/zh/tags)寻找记录稿。"
```

File: i18n/zh.toml [hunk 2]
```diff
@@ -0,0 +1,17 @@
...
+section4_title = "按类别浏览"
+section4_caption= "根据事件的类型和[类别](/categories)搜索记录稿。"
```
Comment (kouloumos) [inline i18n/zh.toml]:
```suggestion
section4_caption= "根据事件的类型和[类别](/zh/categories)搜索记录稿。"
```

**Review by kouloumos — COMMENTED** [review]
A minor change that I missed at my initial review: You need to add the `/zh/` prefix on links in the `zh.toml` file in order to correctly redirect to the translated website.

> As the script, I get a message like this:
> 
> ```
> The most similar command is
>         remote
> Submodule path 'content': checked out 'be8531f274e834c9faa0f8f90f5a4544538f9606'
> ```
> 
> It doesn't return error. It just stays at `bitcointranscripts:master` .

Regarding the script, can you confirm that you are trying to run this from the root folder of your local `bitcointranscripts.github.io` repository? Can you post all the contents of your terminal? Also, what Operating System are you using? Linux, Windows, macOS?

**Comment by editor-Ajian** [comment]
Thanks very much, I accepted these changes and committed them.
Talk about the script. Here is all the contents of my terminal. I am using `Windows 10` and run them in `Git Bash` :
```
Austrian Q@LAPTOP-J68D6PRU MINGW64 /d/Git/bitcointranscripts.github.io (master)
$ git log -3
commit da41b2874aa32df9e69b26492d1794b4250c4e48 (HEAD -> master, origin/master, origin/HEAD)
Author: editor-Ajian <hongji@ethfans.org>
Date:   Sun Oct 30 23:35:47 2022 +0800

    add simplified Chinese

commit 3ac3dd58ca649363ad741e83d2ef4ce73b2c9297
Author: GitHub Actions - content update <actions@github.com>
Date:   Fri Oct 21 12:00:42 2022 +0000

    content update

commit afc84a2103d264b8d02582d5365c24b0d2f5c7b4
Author: GitHub Actions - content update <actions@github.com>
Date:   Tue Oct 18 18:15:34 2022 +0000

    content update

Austrian Q@LAPTOP-J68D6PRU MINGW64 /d/Git/bitcointranscripts.github.io (master)
$ ./preview_branch.sh editor-Ajian master
Previewing branch 'master' of 'https://github.com/editor-Ajian/bitcointranscripts.git':
usage: git submodule [--quiet] add [-b <branch>] [-f|--force] [--name <name>] [--reference <repository>] [--] <repository> [<path>]
   or: git submodule [--quiet] status [--cached] [--recursive] [--] [<path>...]
   or: git submodule [--quiet] init [--] [<path>...]
   or: git submodule [--quiet] deinit [-f|--force] (--all| [--] <path>...)
   or: git submodule [--quiet] update [--init] [--remote] [-N|--no-fetch] [-f|--force] [--checkout|--merge|--rebase] [--[no-]recommend-shallow] [--reference <repository>] [--recursive] [--] [<path>...]
   or: git submodule [--quiet] summary [--cached|--files] [--summary-limit <n>] [commit] [--] [<path>...]
   or: git submodule [--quiet] foreach [--recursive] <command>
   or: git submodule [--quiet] sync [--recursive] [--] [<path>...]
   or: git submodule [--quiet] absorbgitdirs [--] [<path>...]
git: 'restore' is not a git command. See 'git --help'.

The most similar command is
        remote
Submodule path 'content': checked out 'be8531f274e834c9faa0f8f90f5a4544538f9606'
```

**Pull request merged**